source/howto/mobility/getstarted.rst

```diff
@@ -85,24 +85,74 @@ In the Mobility domain, there are three general methods that can be
-#. :literal:`/api/{representation}/{stationTypes}/{dataTypes}`.  In
+#. :literal:`/v2/` gives the list of the Open Data Hub's entry points,
+   that is, the possible representations of the data contained in the
+   datasets. to be used in the next methods. See :ref:`the details
+   below <representation-types>`. 
+
+   .. versionadded:: 2020-June API method to retrieve the list of
+      dataset's entry point.
+
+#. :literal:`/v2/{representation}/` shows all the StationTypes
+   available, that is, all the sources that provided data to the Open
+   Data Hub.
+
+   .. versionadded:: 2020-June API method to retrieve the list of all
+      stationTypes available.
+			
+#. :literal:`/v2/{representation}/{stationTypes}` returns data about
+   the stations themselves, including metadata associated with each, and
+   data about its parent stations, if any.
+#. :literal:`/v2/{representation}/{stationTypes}/{dataTypes}`.  In
    addition to the data of the previous call, it contains the data
-   types defined in the dataset and the most recent measurement. This
-   method is especially suited for real time retrieval of data.
-#. :literal:`/api/{representation}/{stationTypes}/{dataTypes}/{from}/{to}`.
-   All the data retrieved by the previous method, but limited to a
+   types defined in the dataset.
+#. :literal:`/v2/{representation}/{stationTypes}/{dataTypes}/latest`. In
+   addition to all the data retrieved by the previous call, this call
+   retrieves also the most recent measurement. This method is
+   especially suited for real time retrieval of data.
+
+   
+   .. versionadded:: 2020-June API method to retrieve the list of
+      latest/real time measurements
+
+#. :literal:`/v2/{representation}/{stationTypes}/{dataTypes}/{from}/{to}`.
+   All the data retrieved by method #3, but limited to a
    given historical interval (:literal:`from` ... :literal:`to`)
 
-These methods introduce another facility made available by the API v2:
-the type of `representation`: each result set can be reproduced as a
-single, :strong:`flat` or as an indented, :strong:`tree`\-like JSON
-file, the former more suitable for machine consumption, while the
-latter more convenient for human reading.
 
-:literal:`/api/{representation}/{stationTypes}`
+.. _representation-types:
+
+.. topic:: Showing and browsing data with API v2
+	   
+   The first method described in the previous list introduces a new
+   facility made available by the API v2: the type of
+   `representation` that can be used to browse or access the data
+   provided by the Open Data Hub Team.
+
+   The three alternative, which will be described in detail in an
+   upcoming howto, are: `flat`, `tree`, and `apispec`.
+
+   The `flat` and the `tree` alternatives are :term:`JSON`
+   representation of the data and may require to install a JSON plugin
+   to render the output correctly, whereas `apispec` is a YAML
+   representation in OpenAPI v3 format suitable for swagger-like
+   access to the data.
```

Comment:
Would not add browser specifics here, just talk about the API.... I would remove the JSON plugin stuff